lib/logstash/outputs/elasticsearch/http_client.rb

```diff
@@ -176,7 +176,11 @@ def join_bulk_responses(bulk_responses)
     end
 
     def bulk_send(body_stream, batch_actions)
-      params = compression_level? ? {:headers => {"Content-Encoding" => "gzip"}} : {}
+      params = {
+          :query => {"filter_path" => "errors,items.*.error,items.*.status"}
```

Comment:
I wondered about that, and was trying to think of a valid reason why they would change the filter path, other than to filter the response to reduce the payload - this feels like it is an implementation details, which I think we only exposed for the monitoring use case, and the payload reduction we already covered.

Possibly debugging?

Reply:
It looks like manticore (and httpclient) will allow both to coexist:

`Manticore.get("http://localhost:3333/?q=test",  query:  {q: "kittens"}).body`

causes:

```
❯ nc -l 3333           
GET /?q=test&q=kittens HTTP/1.1
Connection: Keep-Alive
Content-Length: 0
Host: localhost:3333
User-Agent: Manticore 0.9.1
Accept-Encoding: gzip,deflate
```

So it depends how ES chooses to treat these cases since the RFC doesn't seem to prohibit this case, nor explain what the server should do.
Either way we should confirm how ES handles this situation and add a note to the docs about how query parameters  in `bulk_path` interact with [`parameters`](https://www.elastic.co/guide/en/logstash/current/plugins-outputs-elasticsearch.html#plugins-outputs-elasticsearch-parameters), w/ a special mention about `filter_path`.

Reply:
I guess we have options

* Keep it as it is in the PR
* Detect `filter_path` in the existing URL and use that instead of the new setting, and log appropriately.
* Detect `filter_path` in the existing URL and drop it, enforcing the new setting, and log appropriately.

I'm easy with any of them, as long as we document it clearly